plugins/katello/3.13/release_notes/release_notes.md

```diff
@@ -26,9 +26,13 @@ Content View descriptions may now be updated via the Hammer CLI.
 
 Katello 3.13 provides the ability to upload SRPMs via the API and through Hammer and Katello to correctly identify the content as a source RPM.
 
+## Change to signing key
+
+Previous releases were signed using the Katelo key, but starting with 3.13 package signing will use the [Foreman Key](https://theforeman.org/static/keys/4674C97BD8C2A3EF63BCB20788BB5C467B5B366A.pub).
```

Comment:
No need to include the domain:
```suggestion
Previous releases were signed using the Katelo key, but starting with 3.13 package signing will use the [Foreman Key](/static/keys/4674C97BD8C2A3EF63BCB20788BB5C467B5B366A.pub).
```

Probably also good to show the fingerprint similar to how we do on https://theforeman.org/security.html